logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/Call.kt

```diff
@@ -16,6 +19,9 @@ data class Call(
     val conversationId: ConversationId,
     val status: CallStatus,
     val callerId: String,
+    val conversationDetails: ConversationDetails,
+    val caller: OtherUser?,
```

Comment:
I think we don't need to pass all of these data here, just the needed values for calling should be enough

Reply:
okay, make sense